src/OpenFeature/Model/EvaluationContextBuilder.cs

```diff
@@ -125,6 +138,23 @@ public EvaluationContextBuilder Set(string key, DateTime value)
         /// <returns>This builder</returns>
         public EvaluationContextBuilder Merge(EvaluationContext context)
         {
+            string newTargetingKey = "";
+
+            if (!string.IsNullOrWhitespace(TargetingKey))
```

Comment:
Ugh, this is my fault, sorry :[

```suggestion
            if (!string.IsNullOrWhiteSpace(TargetingKey))
```